### PR TITLE
feat(user select): added number of investigations

### PR DESCRIPTION
**Pull request by yaniryos**

### CODE REVIEW

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx [hunk 1]
```diff
@@ -66,15 +69,46 @@ const InvestigationTable: React.FC = (): JSX.Element => {
...
             case TableHeadersNames.investigatorName:
                 if (selectedRow === indexedRow.epidemiologyNumber && investigatorAutoCompleteClicked) {
                     return (
                         <Autocomplete
+                            PopperComponent={CustomPopper}
                             test-id='currentInvetigationUser'
-                            options={Array.from(allUsersOfCurrCounty, ([id, value]) => ({id, value}))}
-                            getOptionLabel={(option) => option.value.userName}
+                            options={Array.from(allUsersOfCurrCounty, ([id, value]) => ({ id, value }))}
+                            getOptionLabel={(option) => option.value.userName ? option.value.userName : 'zsfasdf'}
```
Comment (RoeiRom) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx:85]: change 'zsfasdf' to meaningful string

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx [hunk 2]
```diff
@@ -66,15 +69,46 @@ const InvestigationTable: React.FC = (): JSX.Element => {
...
+                            renderOption={(option, { selected }) => (
+                                option.value.userName ?
+                                <React.Fragment
+                                >
```
Comment (RoeiRom) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx:89]: change to <>

**Review by RoeiRom — CHANGES_REQUESTED** [review]

File: client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx [hunk 3]
```diff
@@ -66,15 +69,45 @@ const InvestigationTable: React.FC = (): JSX.Element => {
 
     const user = useSelector<StoreStateType, User>(state => state.user);
 
+    const CustomPopper = (props: any) => {
+        return (<Popper {...props} style={{ width: 350}} placement='bottom-start' />)
```
Comment (roymaniv) [inline client/src/components/App/Content/LandingPage/InvestigationTable/InvestigationTable.tsx:73]: למה בstyle?

File: server/src/DBService/Users/Query.ts [hunk 4]
```diff
@@ -32,9 +32,16 @@ query AllGroupUsers($investigationGroup: Int!) {
       serialNumber
       userName
       investigationGroup
+      newInvestigationsCount: investigationsByLastUpdator(filter: {investigationStatus: {equalTo: "חדשה"}}) {
```
Comment (roymaniv) [inline server/src/DBService/Users/Query.ts:35]: single quotes

File: server/src/DBService/Users/Query.ts [hunk 5]
```diff
@@ -32,9 +32,16 @@ query AllGroupUsers($investigationGroup: Int!) {
...
+        totalCount
+      }
+      activeInvestigationsCount: investigationsByLastUpdator(filter: {investigationStatus: {equalTo: "בטיפול"}}) {
```
Comment (roymaniv) [inline server/src/DBService/Users/Query.ts:38]: single quotes

**Review by roymaniv — APPROVED** [review]

**Review by RoeiRom — APPROVED** [review]

**Pull request merged**